### ISSUE TITLE
Incompatible with RapidLeafDecay mod

### ISSUE THREAD

**Issue by xmesty** [TeamTwilight/twilightforest#1128]
Forge version: 
Twilight Forest version: 1.16.5-4.0.435

Steps to reproduce:
1. Loading the twilight swamp (dark forest area) with the RapidLeafDecay mod

What happened instead: Game crashes every time. I checked multiple times, RapidLeafDecay mod was the issue. I have not tried a different quick-leaf decay mod yet. Rest of Twilight Forest seems fine.

**Comment by GizmoTheMoonPig** [comment]
Send a crashlog so we can check it ourselves

**Comment by xmesty** [comment]
I have a bunch of mods and it kept crashing until I removed RapidLeafDecay yesterday. However, I tried to recreate the issue today, I wanted to give you a version with just TF and RLD, but it wouldn't crash even with all the mods installed, including RLD. So below are a couple of logs from yesterday as I was testing, looks like an issue with Dark Leaves block, but who knows since it isn't crashing anymore. Hope that helps.

[crash-2021-05-16_21.31.01-server.txt](https://github.com/TeamTwilight/twilightforest/files/6497605/crash-2021-05-16_21.31.01-server.txt)
[crash-2021-05-16_21.21.48-server.txt](https://github.com/TeamTwilight/twilightforest/files/6497606/crash-2021-05-16_21.21.48-server.txt)
